class-events-manager-securesubmit.php

```diff
@@ -578,7 +579,19 @@ function authorize_and_capture($EM_Booking){
         //Return transaction_id or false
         return apply_filters('em_gateway_securesubmit_authorize', $result, $EM_Booking, $this);
     }
-
+    /**
+     * set secret key and load service configuration
+     * @return type*
+     */
+    public function setConfig()
+    {
+        $config = new ServicesConfig();
+        $config->secretApiKey = get_option('em_'.$this->gateway.'_secret_key');
+        $config->serviceUrl = "https://cert.api2.heartlandportico.com";
+        $service =  ServicesContainer::configure($config);
+        return $service;    
```

Comment:
The return here isn't necessary. `ServicesContainer::configure` doesn't return anything useful, and the result isn't. necessary elsewhere.

Reply:
updated